src/containers/ProfilePage/ProfilePage.js

```diff
@@ -142,6 +151,50 @@ export const DesktopReviews = props => {
   );
 };
 
+export const CustomUserFields = props => {
+  const { publicData, metadata, userFields } = props;
+  return (
+    <>
+      <H4 as="h2" className={css.listingsTitle}>
```

Comment:
This doesn't sound right (but I'd remove the whole heading as argued in a different comment).

Reply:
Heading removed!